tests/unit/formatters/format-relative-test.ts

```diff
@@ -2,9 +2,9 @@ import { module, test } from 'qunit';
 import FormatRelative from 'ember-intl/-private/formatters/format-relative';
 import { createIntl, OnErrorFn } from '@formatjs/intl';
 
-function getIntl(locales: string | string[]) {
+function getIntl(locale: string) {
   return createIntl({
-    locale: Array.isArray(locales) ? locales[0] : locales,
+    locale: locale,
```

Comment:
`locales` was never an array, so I just changed this to a `string`

Reply:
Not essential but

```suggestion
    locale,
```